.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build and Push
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: praekeltfoundation/malaria24-django
+          tag_with_ref: true
+          tag_with_sha: true
```

Comment:
please add a new line here. It's conventional practise to always have a new line at the end of a file.